### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by faze-quadrata**
## Checklist
1. [ ] Unit Testing?
2. [ ] Backward Compatible?
3. [ ] Upgrade Safe (storage)?
4. [ ] Updated Confluence?
5. [ ] Updated Gitbook?

### CODE REVIEW

File: contracts/QuadPassport.sol [hunk 1]
```diff
@@ -269,6 +269,7 @@ contract QuadPassport is IQuadPassport, UUPSUpgradeable, QuadSoulbound, QuadPass
         if (isEmpty){
             _burnPassports(_account);
         }
+        emit BurnPassportsIssuer(_msgSender(), _account);
```
Comment (faze-quadrata) [inline contracts/QuadPassport.sol:272]: Emit this log since we dont always burn passports

File: test/QuadPassport/test_burn_passport.ts [hunk 2]
```diff
@@ -1838,10 +1866,237 @@ describe("QuadPassport", async () => {
...
+        .attributes(minterA.address, ATTRIBUTE_IS_BUSINESS);
+      expect(isBusinessAttributesMid.length).equals(1);
+      const isBusinessMidBurnA = isBusinessAttributesMid.find(
+        (attr: any) => attr.issuer == issuer.address
+      );
+      const isBusinessMidBurnB = isBusinessAttributesMid.find(
+        (attr: any) => attr.issuer == issuerB.address
+      );
+
+      expect(amlMidBurnA.value).equals(attributes[ATTRIBUTE_AML]);
```
Comment (faze-quadrata) [inline test/QuadPassport/test_burn_passport.ts:2017]: Note that AML for both issuers can still be purchased even if one issuer called burn function

File: contracts/storage/QuadGovernanceStore.sol [hunk 3]
```diff
@@ -25,7 +25,7 @@ contract QuadGovernanceStore is QuadConstant {
 
     // Others
     uint256 internal _revSplitIssuer; // 50 means 50%;
-    uint256 internal _maxEligibleTokenId;
+    uint256 public _maxEligibleTokenId;
```
Comment (fabrice-quadrata) [inline contracts/storage/QuadGovernanceStore.sol:28]: won't you run into a problem with this?

I remember @teddy-quadrata made all attributes internal for a reason, but cannot remmeber why.

Comment (faze-quadrata) [reply contracts/storage/QuadGovernanceStore.sol:28]: yeah forgot i already have a function for it

**Review by fabrice-quadrata — APPROVED** [review]

**Pull request merged**